data-processing/scripts/run_pipeline.py

```diff
@@ -335,8 +338,29 @@ def run_commands_for_arxiv_ids(
         type=str,
         help="Base url for definition model paths."
     )
+
+    parser.add_argument(
+        "--output-form",
+        type=str,
+        choices=[c.value for c in OutputForm],
+        default=OutputForm.DB.value,
+        help=(
+            f"How we want the output to be stored. "
+            + "If you specify file or both, you must also provide a value for output-dir."
+        )
+    )
+
+    parser.add_argument(
+        "--output-dir",
+        type=str,
+        help="The directory in which to write the outputs."
+    )
+
     args = parser.parse_args()
 
+    # check that output arguments make sense
+    OutputDetails.validate(output_form=args.output_form, output_dir=args.output_dir)
```

Comment:
Upon assertion failure, it might make sense to throw an error using Python's [`parser.error`](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.error) method if validation fails. Though this is a minor point and not that important.

Reply:
Fine with me! Now:
```
python scripts/run_pipeline.py --one-paper-at-a-time --arxiv-ids "1601.00978v1" --output-form file  --entities citations
wandb: WARNING W&B installed but not logged in.  Run `wandb login` or set the WANDB_API_KEY env variable.
usage: run_pipeline.py [-h] [-v] [--config CONFIG] [--log-prefix LOG_PREFIX]
                       [--entities {sentences-pdf,sentences,symbols,definitions,citations,equations,glossary-terms} [{sentences-pdf,sentences,symbols,definitions,citations,equations,glossary-terms} ...]]
                       [--commands {fetch-arxiv-sources,fetch-s2-metadata,unpack-sources,compile-tex,normalize-tex,compile-normalized-tex,fetch-arxiv-pdf,raster-pages,detect-sentences,extract-bibitems,resolve-bibitems,locate-bounding-boxes-for-citation-fragments,locate-citations,upload-citations,detect-equations,locate-bounding-boxes-for-equations,upload-equations,detect-glossary-terms,extract-contexts-for-glossary-terms,locate-bounding-boxes-for-glossary-terms,upload-glossary-terms,extract-symbols,find-symbol-matches,extract-contexts-for-symbols,locate-bounding-boxes-for-equation-tokens,locate-bounding-boxes-for-symbols-with-affixes,locate-composite-symbols,collect-symbol-locations,upload-symbols,fetch-spp-jsons,locate-sentences,upload-sentences,tokenize-sentences,create-annotation-files,detect-definitions,extract-contexts-for-definitions,locate-bounding-boxes-for-definitions,upload-definitions} [{fetch-arxiv-sources,fetch-s2-metadata,unpack-sources,compile-tex,normalize-tex,compile-normalized-tex,fetch-arxiv-pdf,raster-pages,detect-sentences,extract-bibitems,resolve-bibitems,locate-bounding-boxes-for-citation-fragments,locate-citations,upload-citations,detect-equations,locate-bounding-boxes-for-equations,upload-equations,detect-glossary-terms,extract-contexts-for-glossary-terms,locate-bounding-boxes-for-glossary-terms,upload-glossary-terms,extract-symbols,find-symbol-matches,extract-contexts-for-symbols,locate-bounding-boxes-for-equation-tokens,locate-bounding-boxes-for-symbols-with-affixes,locate-composite-symbols,collect-symbol-locations,upload-symbols,fetch-spp-jsons,locate-sentences,upload-sentences,tokenize-sentences,create-annotation-files,detect-definitions,extract-contexts-for-definitions,locate-bounding-boxes-for-definitions,upload-definitions} ...]]
                       [--start {fetch-arxiv-sources,fetch-s2-metadata,unpack-sources,compile-tex,normalize-tex,compile-normalized-tex,fetch-arxiv-pdf,raster-pages,detect-sentences,extract-bibitems,resolve-bibitems,locate-bounding-boxes-for-citation-fragments,locate-citations,upload-citations,detect-equations,locate-bounding-boxes-for-equations,upload-equations,detect-glossary-terms,extract-contexts-for-glossary-terms,locate-bounding-boxes-for-glossary-terms,upload-glossary-terms,extract-symbols,find-symbol-matches,extract-contexts-for-symbols,locate-bounding-boxes-for-equation-tokens,locate-bounding-boxes-for-symbols-with-affixes,locate-composite-symbols,collect-symbol-locations,upload-symbols,fetch-spp-jsons,locate-sentences,upload-sentences,tokenize-sentences,create-annotation-files,detect-definitions,extract-contexts-for-definitions,locate-bounding-boxes-for-definitions,upload-definitions}]
                       [--end {fetch-arxiv-sources,fetch-s2-metadata,unpack-sources,compile-tex,normalize-tex,compile-normalized-tex,fetch-arxiv-pdf,raster-pages,detect-sentences,extract-bibitems,resolve-bibitems,locate-bounding-boxes-for-citation-fragments,locate-citations,upload-citations,detect-equations,locate-bounding-boxes-for-equations,upload-equations,detect-glossary-terms,extract-contexts-for-glossary-terms,locate-bounding-boxes-for-glossary-terms,upload-glossary-terms,extract-symbols,find-symbol-matches,extract-contexts-for-symbols,locate-bounding-boxes-for-equation-tokens,locate-bounding-boxes-for-symbols-with-affixes,locate-composite-symbols,collect-symbol-locations,upload-symbols,fetch-spp-jsons,locate-sentences,upload-sentences,tokenize-sentences,create-annotation-files,detect-definitions,extract-contexts-for-definitions,locate-bounding-boxes-for-definitions,upload-definitions}]
                       [--arxiv-ids ARXIV_IDS [ARXIV_IDS ...]]
                       [--arxiv-ids-file ARXIV_IDS_FILE] [--days DAYS]
                       [--s3-job-spec S3_JOB_SPEC] [--source {arxiv,s3}]
                       [--s3-arxiv-sources-bucket S3_ARXIV_SOURCES_BUCKET]
                       [--entity-batch-size ENTITY_BATCH_SIZE]
                       [--keep-intermediate-files] [--max-papers MAX_PAPERS]
                       [--one-paper-at-a-time] [--keep-paper-data]
                       [--store-results] [--store-log]
                       [--s3-output-bucket S3_OUTPUT_BUCKET]
                       [--notify-emails NOTIFY_EMAILS [NOTIFY_EMAILS ...]]
                       [--notify-bcc NOTIFY_BCC [NOTIFY_BCC ...]]
                       [--extraction-only] [--dry-run]
                       [--database-schema DATABASE_SCHEMA]
                       [--database-create-tables]
                       [--data-version DATA_VERSION]
                       [--definitions-base-url DEFINITIONS_BASE_URL]
                       [--output-forms {db,file} [{db,file} ...]]
                       [--output-dir OUTPUT_DIR]
run_pipeline.py: error: Please include either both 'file' as an output form and an output dir, or neither.
```